ironmon_tracker/Constants.lua

```diff
@@ -494,7 +494,8 @@ Constants.CharWidths = {
     ["9"] = 4,
     ["="] = 4,
     [","] = 2,
-    ["-"] = 2,
+	["-"] = 2,
+	["_"] = 5,
```

Comment:
Looks like you replaced tabs with spaces. Can you revert this back to make sure it's using tab indents only?

Reply:
The indentation in that file was/is using spaces for some values in that table. I guess I accidentally converted the indentation for the "-" entry to a tab via my IDE
![image](https://github.com/besteon/Ironmon-Tracker/assets/19582161/8959c797-3439-431a-871d-33d5ddf1a1d4)

Reply:
Oh that's strange. I wonder how long it's been like that.

I'll make this update on dev later then, don't worry about this. Thanks for noticing it.